tests/StateSnapshot.test.js

```diff
@@ -70,6 +70,21 @@ function useProxyStateExample(a) {
       a.a = 'example';
   }, [s.a]);
 }`,
+    `
+const state = proxy({ count: 0 })
+
+      function App() {
+        const snap = useSnapshot(state)
+        const handleClick = useCallback(() => {
+          console.log(snap.count) // This is not recommended as it can be stale.
```

Comment:
```suggestion
          console.log(snap.count)
```

Reply:
i left it in the valid ones.....
one sec